src/content/docs/security/security-privacy/compliance/data-encryption.mdx

```diff
@@ -174,34 +112,14 @@ New Relic's account-level encryption at rest allows approved New Relic customers
       </td>
 
       <td>
-        Account-level encryption includes:
-
+        * [Log management](/docs/logs/new-relic-logs/get-started/new-relic-logs-security-privacy) data
+        * [Dimensional metrics](/docs/using-new-relic/data/understand-data/new-relic-data-types#metric-time-series) data
         * [Log management](/docs/logs/new-relic-logs/get-started/new-relic-logs-security-privacy) data
         * [Dimensional metrics](/docs/using-new-relic/data/understand-data/new-relic-data-types#metric-time-series) used by Telemetry SDKs, Metric API, some integrations
         * [Events](/docs/using-new-relic/data/understand-data/new-relic-data-types#event-data)
         * [Distributed traces](/docs/understand-dependencies/distributed-tracing/get-started/introduction-distributed-tracing)
         * Backups for all data with account-level encryption
```

Comment:
Do we need to drop the reference to account-level encryption here?

Reply:
Yes. That is kind of the point of this: to drop the language that distinguishes between types of encryption at rest.